lld/ELF/ScriptLexer.h

```diff
@@ -9,23 +9,33 @@
 #ifndef LLD_ELF_SCRIPT_LEXER_H
 #define LLD_ELF_SCRIPT_LEXER_H
 
+#include "ScriptToken.h"
 #include "lld/Common/LLVM.h"
+#include "llvm/ADT/StringMap.h"
 #include "llvm/ADT/StringRef.h"
 #include "llvm/Support/MemoryBufferRef.h"
 #include <vector>
 
 namespace lld::elf {
-
 class ScriptLexer {
 public:
+  static const llvm::StringMap<Tok> keywordTokMap;
+  struct Token {
+    Tok kind;
+    StringRef val;
+    inline bool operator==(StringRef other) { return val == other; }
```

Comment:
`inline` is redundant here and below; function definitions within class definitions are implicitly inline.